tools/deployment/make_linux_package.sh

```diff
@@ -51,7 +52,6 @@ DEBUG_PREFIX=$WORKING_DIR/debug
 
 function usage() {
   fatal "Usage: $0 -t deb|rpm -i REVISION -d DEPENDENCY_LIST
-
```

Comment:
After rebase squashing your commits from master here is the effective patch:
```diff
diff --git a/tools/deployment/make_linux_package.sh b/tools/deployment/make_linux_package.sh
old mode 100755
new mode 100644
index b2d8c1a..6db5765
--- a/tools/deployment/make_linux_package.sh
+++ b/tools/deployment/make_linux_package.sh
@@ -34,6 +34,7 @@ SYSTEMD_SERVICE_SRC="$SCRIPT_DIR/osqueryd.service"
 SYSTEMD_SERVICE_DST="/usr/lib/systemd/system/osqueryd.service"
 SYSTEMD_SYSCONFIG_SRC="$SCRIPT_DIR/osqueryd.sysconfig"
 SYSTEMD_SYSCONFIG_DST="/etc/sysconfig/osqueryd"
+SYSTEMD_SYSCONFIG_DST_DEBIAN="/etc/default/osqueryd"
 CTL_SRC="$SCRIPT_DIR/osqueryctl"
 PACKS_SRC="$SOURCE_DIR/packs"
 PACKS_DST="/usr/share/osquery/packs/"
@@ -51,7 +52,7 @@ DEBUG_PREFIX=$WORKING_DIR/debug
 
 function usage() {
   fatal "Usage: $0 -t deb|rpm -i REVISION -d DEPENDENCY_LIST
-
+  
   This will generate an Linux package with:
   (1) An example config /var/osquery/osquery.example.config
   (2) An init.d script /etc/init.d/osqueryd
@@ -120,11 +121,16 @@ function main() {
   cp $OSQUERY_EXAMPLE_CONFIG_SRC $INSTALL_PREFIX$OSQUERY_EXAMPLE_CONFIG_DST
   cp $PACKS_SRC/* $INSTALL_PREFIX/$PACKS_DST
 
-  if [ $OSQUERY_CONFIG_SRC != "" ] && [ -f $OSQUERY_CONFIG_SRC ]; then
+  if [[ $OSQUERY_CONFIG_SRC != "" ]] && [[ -f $OSQUERY_CONFIG_SRC ]]; then
     log "config setup"
     cp $OSQUERY_CONFIG_SRC $INSTALL_PREFIX/$OSQUERY_ETC_DIR/osquery.conf
   fi
-
+  
+  if [[ $DISTRO = "xenial" ]]; then
+    #Change config path to Ubuntu/Xenial default
+    SYSTEMD_SYSCONFIG_DST=$SYSTEMD_SYSCONFIG_DST_DEBIAN
+  fi 
+     
   if [[ $DISTRO = "centos7" || $DISTRO = "rhel7" || $DISTRO = "xenial" ]]; then
     # Install the systemd service and sysconfig
     mkdir -p `dirname $INSTALL_PREFIX$SYSTEMD_SERVICE_DST`
@@ -135,6 +141,11 @@ function main() {
     mkdir -p `dirname $INSTALL_PREFIX$INITD_DST`
     cp $INITD_SRC $INSTALL_PREFIX$INITD_DST
   fi
+  
+  if [[ $DISTRO = "xenial" ]]; then
+    #Change config path in service unit
+    sed -i 's/sysconfig/default/g' $INSTALL_PREFIX$SYSTEMD_SERVICE_DST
+  fi
 
   log "creating package"
   IFS=',' read -a deps <<< "$PACKAGE_DEPENDENCIES"
@@ -153,7 +164,7 @@ function main() {
   fi
 
   POSTINST_CMD=""
-  if [ $OSQUERY_POSTINSTALL != "" ] && [ -f $OSQUERY_POSTINSTALL ]; then
+  if [[ $OSQUERY_POSTINSTALL != "" ]] && [[ -f $OSQUERY_POSTINSTALL ]]; then
     POSTINST_CMD="--after-install $OSQUERY_POSTINSTALL"
   fi
 ```